tests/automation_framework/src/libs/avalon_test_wrapper.py

```diff
@@ -62,7 +62,7 @@ def build_request_obj(input_json_obj,
     worker_lookup SDK function requires worker_type parameter
     worker_retrieve SDK function requires worker_id parameter.
     """
-    action_obj = eval(input_json_obj["method"]+"()")
+    action_obj = eval(input_json_obj.get("method")+"()")
```

Comment:
what is this change for?

Reply:
In case where method field doesn't exist our framework should not throw error